### PR TITLE
Use black instead of "flake8-black" on CI.

### PR DESCRIPTION
**Pull request by penguinolog**
- Use `black` for code formatting and validate using `black --check`. Code compatibility: py26-py39.
- Enforce maximal line length to 120 symbols

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[build-system]
```
Comment (penguinolog) [inline pyproject.toml:1]: Unfortunatelly without `build-system` will be complaints from `pip`

File: setup.cfg [hunk 2]
```diff
@@ -5,8 +5,8 @@ url = https://github.com/jd/tenacity
 summary = Retry code until it succeeds
 long_description = Tenacity is a general-purpose retrying library to simplify the task of adding retry behavior to just about anything.
 author = Julien Danjou
-author-email = julien@danjou.info
```
Comment (penguinolog) [inline setup.cfg:8]: dash separated fields is deprecated

**Comment by penguinolog** [comment]
Python code was automatically formatted by `black` (no manual edit except removal of comments for `pep8-black`)

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+# Minimum requirements for the build system to execute.
+# PEP 508 specifications for PEP 518.
+# Banned setuptools versions have well-known issues
+requires = [
+  "setuptools >= 21.0.0,!=24.0.0,!=34.0.0,!=34.0.1,!=34.0.2,!=34.0.3,!=34.1.0,!=34.1.1,!=34.2.0,!=34.3.0,!=34.3.1,!=34.3.2,!=36.2.0",  # PSF/ZPL
```
Comment (jd) [inline pyproject.toml:6]: not a fan of this list…
obviously all version have issues or they wouldn't be new releases lol

**Review by jd — APPROVED** [review]

**Pull request merged**